### ISSUE TITLE
AGP 21:  Do you like RocketChat as a chat platform for the Aragon community?

### ISSUE THREAD

**Issue by luisivan** [aragon/governance#30]
Do you like RocketChat as a chat platform for the Aragon community?
- Yes
- It's okay
- No

**Comment by yutarond** [comment]
No. Slack is better.

**Comment by john-light** [comment]
We will not be going back to Slack.

https://github.com/aragon/governance/blob/master/AGP-5_Migration_to_Open_Source_Messaging_Platform.md

**Comment by ntquangbk** [comment]
It's ok, but i think Telegram is more popular, more convenient (no need to login multiple projects), Telegram is good for marketing in the early stage and Rocket chat is better when Aragon is popular.

**Comment by MeanderingCode** [comment]
Has anyone considered the use of Matrix? It is a decentralized chat protocol with federation at the core and bridges to many other platforms. It shares many values with many blockchain projects, and several use it as their chat system (Tezos and Polkadot, to name two).

More information can be found at https://matrix.org and an open hosted server for a web frontend can be found at https://riot.im.

I would vote for moving to Matrix.

**Comment by luisivan** [comment]
@MeanderingCode yeah, we thought about it back in the day, see https://github.com/aragon/governance/tree/master/AGP-5_Migration_from_Slack

**Comment by john-light** [comment]
@MeanderingCode Thank you for the suggestion! I would be in favor of creating bridges between our Rocket Chat instance and Matrix (as we have done with the Aragon DAC channels) but I think a full migration would be too much to handle right now, especially since we are still recovering from our migration off of Slack last year.

@ntquangbk Telegram is a centralized app with little to no support for public community group administrators. It is too easy for bots, spammers, and scammers to ruin the experience for a community. We will not be switching to that platform any time soon.

**Comment by MeanderingCode** [comment]
I understand not wanting to take on yet another migration in a short amount of time.

That said, I highly prefer Matrix to RocketChat, for a variety of reasons. The top three are features and UX, federation and bridging, and the community and mission.

If this survey is not about moving, is it just about temperature gauging? What happens if the satisfaction is low?
